styles/styles.css

```diff
@@ -712,10 +712,16 @@ main .section.thank-you p {
 }
 
 /* section - light-blue */
-main .section.light-blue {
+main .section.light-blue,
+main .section.light-blue .section-container {
   background-color: var(--transparent-blue-light-color);
 }
 
+/* section - padded */
+main .section.padded .section-container {
```

Comment:
Since we are adding a variant here, please add in the block library if not done already.

Reply:
Yep, I usually add those at the end

Reply:
Done 👍